src/dcos_e2e/_common.py

```diff
@@ -7,40 +7,7 @@
 from subprocess import PIPE, STDOUT, CompletedProcess, Popen
 from typing import Dict, List, Optional, Union
 
-
-def get_logger(name: str) -> logging.Logger:
-    """
-    Return a customized logger that is able to log on DEBUG level.
-
-    Args:
-        name: Name of the logger. Setting this to ``__name__`` will log
-            the path to the file where the logger has been created.
-
-    Returns:
-        See :py:class:`logging.Logger`.
-    """
-    # This gets the root logger with a ``logging.lastResort``
-    # StreamHandler that logs on WARNING level.
-    logger = logging.getLogger(name=name)
-    logger.setLevel(logging.DEBUG)
```

Comment:
Will a test suite which uses DC/OS E2E now have to do something in order to see useful information - something it did not have to do before?

If so, can that go in the CHANGELOG please.